pyne/CMakeLists.txt

```diff
@@ -152,4 +152,20 @@ if(BUILD_SPATIAL_SOLVER)
         DEPENDS pyne
         )
     target_link_libraries(transport_spatial_methods pyne)
+
+    #add_executable(alphad ${ENSDF_PROCESSING_ALPHAD_SRC})
+    add_executable(alphad ${PROJECT_SOURCE_DIR}/src/ensdf_processing/ALPHAD/alphad.f)
```

Comment:
This probably won't work on its own, because we never run `cmake install`.  Something needs to move the built executable to the top-level `scripts/` directory so that setup.py knows to install it.